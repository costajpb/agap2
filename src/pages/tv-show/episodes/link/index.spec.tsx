import { render } from "@testing-library/react"
import Link from "."

describe('pages/tv-show/episodes/link', () => {
    const episode = {
        id: 1,
        title: 'Dummy title'
    }

    it('should render without image', () => {
        const { container } = render(<Link episode={episode} />)
        expect(container).toMatchSnapshot()
    })

    it('should render with image', () => {
        const { container } = render(<Link episode={{
            ...episode,
            coverImage: 'image-src'
        }} />)
        expect(container).toMatchSnapshot()
    })
})