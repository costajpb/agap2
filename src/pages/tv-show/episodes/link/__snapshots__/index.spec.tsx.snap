// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/tv-show/episodes/link should render with image 1`] = `
<div>
  <a
    class="sc-aXZVg hEUmpn"
    data-episode-id="1"
    data-testid="episode"
    href="/episodes/1"
  >
    <img
      alt="Dummy title"
      class="sc-fBWQRz jOjvwV"
      src="image-src"
    />
    <span
      class="sc-kMkxaj gmRHkp"
    >
      Dummy title
    </span>
  </a>
</div>
`;

exports[`pages/tv-show/episodes/link should render without image 1`] = `
<div>
  <a
    class="sc-aXZVg hEUmpn"
    data-episode-id="1"
    data-testid="episode"
    href="/episodes/1"
  >
    <img
      alt="Dummy title"
      class="sc-fBWQRz jOjvwV"
    />
    <span
      class="sc-kMkxaj gmRHkp"
    >
      Dummy title
    </span>
  </a>
</div>
`;
